### ISSUE TITLE
Error importing YAMLDumper

### ISSUE THREAD

**Issue by ChrisTerBeke** [marshmallow-code/apispec#388]
As of last week, we are suddenly getting this line:

```
ImportError: cannot import name 'YAMLDumper' from 'apispec.core' (/usr/local/lib/python3.7/site-packages/apispec/core.py)
```

Seems like YAMLDumper was moved to another module which is a breaking change, but the version increase was not a major one?

**Comment by lafrech** [comment]
We released apispec 1.0.0 just recently. It is a major version with a lot of breaking changes documented in the changelog.

I think your issue is related to 

https://github.com/marshmallow-code/apispec/blob/dev/CHANGELOG.rst#100b1-2018-07-29

https://github.com/marshmallow-code/apispec/pull/251

I forgot to link to the PR in the CHANGELOG. ~I shall fix that.~ [Fixed](https://github.com/marshmallow-code/apispec/commit/ccab49f804258428652d1130cabdcc040cd5d6fb).

**Comment by ChrisTerBeke** [comment]
Okay, thanks for the clarification. I'll use the last beta then for now and upgrade in the future after rewriting some of our code.
